src/containers/app/index.js

```diff
@@ -6,17 +6,19 @@ import Body from '../body'
 import Header from '../header'
 import Sidebar from '../sidebar'
 import Chat from '../chat'
+import RegisterStepTwo from '../registerStepTwo'
 
 const App = () => (
     <div>
         <main>

```

Comment:
Is main a component you make? Strange I don't see it bweing imported, also usually the components are first letter uppercase.

Reply:
I changed it to be a div, as we don't have a "main" component.